.env.example

```diff
@@ -1,3 +1,10 @@
-ACCESS_KEY_ID=
-SECRET_ACCESS_KEY=
-BUCKET_NAME=
+ACCESS_KEY_ID=dummy key
```

Comment:
I'd really like to have some comments in here, detailing what the significance is of these vars, and where their values come from